example/build.gradle.kts

```diff
@@ -14,13 +14,40 @@ kotlin {
     }
 
     spm {
-        ios("11") {
+        ios("13") {
             dependencies {
                 packages(
                     url = "https://github.com/AFNetworking/AFNetworking.git",
-                    version = "4.0.0",
+                    version = "4.0.1",
                     name = "AFNetworking"
                 )
+                /* TODO: Support building target dependencies? i.e `FirebaseCore` & `FirebaseSharedSwift`
```

Comment:
Packages which have dependencies such as `FirebaseRemoteConfig` we will need to loop through the dependencies defined in the FIrebase `Package.swift`. Unless there is another way to identify the dependencies.